FirstSolution/Part16Task1/ISwimable.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Part16Task1
+{
+    public interface ISwimable
+    {
+        void Swim()
+        {
+        }
```

Comment:
Remove brackets, they need to specify default realization

Reply:
Done